src/Sarif/ConsoleLogger.cs

```diff
@@ -136,10 +143,16 @@ private void WriteToConsole(ResultLevel level, Uri uri, Region region, string ru
             Region region,
             string ruleId,
             string message,
-            ResultLevel resultLevel)
+            FailureLevel resultLevel,
```

Comment:
>resultLevel [](start = 25, length = 11)

I would either consistently name these variables `level` and `kind` (as you did on L104), or consistently name them for their types (`failureLevel` and `resultKind`).

Reply:
I prefer the latter. We have too many "kinds" (and possibly "levels", I don't remember) floating around.

---
In reply to: [252893421](https://github.com/Microsoft/sarif-sdk/pull/1249#discussion_r252893421) [](ancestors = 252893421)